src/map/spell.h

```diff
@@ -234,6 +234,18 @@ enum SPELLFLAG
     SPELLFLAG_IGNORE_SHADOWS = 0x04  // Ignore shadows and hit player anyways (example: Mobs "Death" spell)
 };
 
+enum SPELLTYPE
+{
+    SPELLTYPE_NONE   = 0x00,
+    SPELLTYPE_HEAL   = 0x01,
+    SPELLTYPE_CURE   = 0x02,
+    SPELLTYPE_BUFF   = 0x04,
+    SPELLTYPE_DEBUFF = 0x08,
+    SPELLTYPE_NA     = 0x10,
+    SPELLTYPE_RAISE  = 0x20,
+    SPELLTYPE_SEVERE = 0x40,
+};
```

Comment:
Whats the difference between heal and cure?

Reply:
Heal is healing magic, in general, ok

Cure is cure and cura, families... Why not use families then?